TLM/TLM/Manager/Impl/LaneArrowManager.cs

```diff
@@ -218,5 +220,238 @@ public class LaneArrowManager
         public static ICustomDataManager<List<Configuration.LaneArrowData>> AsLaneArrowsDM() {
             return Instance;
         }
+
+        public static class SeparateTurningLanes{
```

Comment:
One class per `.cs` file :)

Reply:
I suggested wrapping it with a class. It has little sense to exist in a separate file, unless that file name will begin with `LaneArrowManager*`, but that's one nice way to do it indeed.

Reply:
Valid point @kvakvs , ignore my previous comment 😃